standard/docs/en/schema/reference.md

```diff
@@ -141,10 +146,7 @@ It may contain details of a forthcoming process to receive and evaluate proposal
 
 ### Bids
 
-```eval_rst
-.. extensionlist:: The optional bids extension can be used to provide summary and detailed information about bids.
-   :list: bids
-```
+The optional [Bid statistics and details](http://extensions.open-contracting.org/en/extensions/bids/) extension can be used to provide bid statistics and detailed information about individual bids.
```

Comment:
Should this be a link to the correct version of the extension for this OCDS release? (the current link redirects to the master branch)

Reply:
Done in 127b5d4